hudi-cli/src/test/java/org/apache/hudi/cli/commands/AbstractShellIntegrationTest.java

```diff
@@ -58,4 +58,13 @@ public void teardown() throws Exception {
   protected static JLineShellComponent getShell() {
     return shell;
   }
-}
\ No newline at end of file
+
+  /**
+   * Helper to prepare string for matching.
+   * @param str Input string.
+   * @return pruned string with non word characters removed.
```

Comment:
this method is not used only. This are used by TestClasses which are not added to 0.5.3.